nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -3,57 +3,62 @@
 with lib;
 
 let
+  settingsFormat = let
```

Comment:
The module looks good to me, but @infinisil may have interesting feedback on it.